share/dictionary/dhcpv4/dictionary.rfc3495

```diff
@@ -0,0 +1,15 @@
+# -*- text -*-
+# Copyright (C) 2022 The FreeRADIUS Server project and contributors
+# This work is licensed under CC-BY version 4.0 https://creativecommons.org/licenses/by/4.0
+# Version $Id$
+##############################################################################
+#
+#  RFC3495 - Dynamic Host Configuration Protocol (DHCP) Option for
+#            CableLabs Client Configuration
+#
+#  $Id$
+#
+##############################################################################
+
+# CableLabs Client Configuration
+ATTRIBUTE	CCC					122	octets
```

Comment:
this should be `tlv`, with sub-options as defined in section 4.

Reply:
Done.